gwt-editor-processor/pom.xml

```diff
@@ -232,6 +234,11 @@
                 <groupId>org.codehaus.mojo</groupId>
                 <artifactId>flatten-maven-plugin</artifactId>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-deploy-plugin</artifactId>
+                <version>${maven.deploy.plugin}</version>
```

Comment:
need to specify skip=false here to override parent, right?

Reply:
oh yes!